scaletest/workspacetraffic/metrics.go

```diff
@@ -0,0 +1,49 @@
+package workspacetraffic
+
+import "github.com/prometheus/client_golang/prometheus"
+
+type Metrics struct {
+	BytesRead      prometheus.CounterVec
+	BytesWritten   prometheus.CounterVec
+	Errors         prometheus.CounterVec
+	ReadLatencyMS  prometheus.HistogramVec
+	WriteLatencyMS prometheus.HistogramVec
+	LabelNames     []string
+}
+
+func NewMetrics(reg prometheus.Registerer, labelNames ...string) *Metrics {
+	m := &Metrics{
+		BytesRead: *prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: "coderd",
+			Subsystem: "scaletest",
+			Name:      "bytes_read",
+		}, labelNames),
+		BytesWritten: *prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: "coderd",
+			Subsystem: "scaletest",
+			Name:      "bytes_written",
+		}, labelNames),
+		Errors: *prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: "coderd",
+			Subsystem: "scaletest",
+			Name:      "errors",
+		}, labelNames),
+		ReadLatencyMS: *prometheus.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
ReadLatencyMS vs `read_latency_seconds` - is it seconds or milliseconds?

same for WriteLatencyMS